apps/server/src/modules/school/uc/school.uc.ts

```diff
@@ -27,6 +27,18 @@ export class SchoolUc {
 			action: Actions.read,
 			requiredPermissions: [Permission.SCHOOL_EDIT],
 		});
+		const school: SchoolDO = await this.schoolService.getSchoolById(schoolId);
+
+		const shouldRestartMigration =
+			!!school.oauthMigrationFinished &&
```

Comment:
Which type is school.oauthMigrationFinished and !!school.oauthMigrationMandatory ?

Can you please outsource the condtion in private method?

Reply:
Will be done with my the comment from sevenways